lib/matplotlib/tests/test_axes.py

```diff
@@ -8272,3 +8272,15 @@ def test_bar_all_nan(fig_test, fig_ref):
 
     ax_ref.bar([1], [1]).remove()
     ax_ref.bar([1], [1])
+
+
+@image_comparison(["extent_units.pdf"], style="mpl20")
+def test_extent_units():
+    mpl.style.use("mpl20")
+    _, ax = plt.subplots()
+    dateFirst = np.datetime64('2020-01-01', 'D')
+    dateLast = np.datetime64('2020-01-11', 'D')
```

Comment:
Also should be using `snake_case`, not `camelCase`.
```suggestion
    date_first = np.datetime64('2020-01-01', 'D')
    date_last = np.datetime64('2020-01-11', 'D')
```

Reply:
done